src/libs/ajax/workflows-app/Cbas.js

```diff
@@ -19,6 +19,11 @@ export const Cbas = (signal) => ({
       const res = await fetchFromProxy(cbasUrlRoot)('api/batch/v1/methods', _.mergeAll([authOpts(), jsonBody(payload), { signal, method: 'POST' }]));
       return res.json();
     },
+    delete: async (cbasUrlRoot, methodId) => {
```

Comment:
TOL: I know we're saying "delete" in the UI but it might be nice on the "behind the scenes" side to call this archive?

Reply:
yeah -- I called it "delete" to match the http code, but I think you're right